cpp/src/parquet/arrow/schema.h

```diff
@@ -163,24 +165,28 @@ struct PARQUET_EXPORT SchemaManifest {
     return it->second;
   }
 
-  bool GetFieldIndices(const std::vector<int>& column_indices, std::vector<int>* out) {
+  ::arrow::Result<std::vector<int>> GetFieldIndices(
+      const std::vector<int>& column_indices) {
     // Coalesce a list of schema field indices which are the roots of the
```

Comment:
could we move the comment to a docstring and give an example?

Reply:
will do